mscore/menus.cpp

```diff
@@ -1595,7 +1595,9 @@ PalettePanel* MuseScore::newTextPalettePanel(bool defaultPalettePanel)
       stxt = new SystemText(gscore, Tid::TEMPO);
       /*: System text to switch from swing rhythm back to straight rhythm */
       stxt->setXmlText(QT_TRANSLATE_NOOP("Palette", "Straight"));
-      stxt->setSwing(false); // redundant, being the default anyhow, but for documentation
+      // need to be true to enable the "Off" option
+      stxt->setSwing(true);
+      stxt->setSwingParameters(0, stxt->score()->styleI(Sid::swingRatio));
```

Comment:
Shouldn't this be a ratio of 1:1 instead?

Reply:
Might be better to leave it set to a 'normal' swing ratio, but off., so switching it on works?

Reply:
Although indeed a swing ratio of 1:1, or rather `50` should work too for this particular text

Reply:
This is what a staff text with no swing settings have. For example if you add a pizz. text, the swing tab will appear so.

Reply:
Yes, but question is whethwer for a "Straight" text it may make sense to set Swing off and to 50:50

Reply:
I don't think so, because it should have the same values of texts which have swing settings disabled.

Reply:
It the straight text from the palettes set up correctly?
It should be set up the same way.

Reply:
This PR _is_ about the Straight text from the palettes.

Reply:
Straight means 50:50. Or off, both have the same result

Reply:
No, it is the one from the master palette

Reply:
not from Basic,xml or Advanced.xml

Reply:
Ah, I see, those don't have any such element

Reply:
BTW, that bug looks very much like a Brown Bag for me ;-)

Reply:
Ultimately it doesn't matter, but somehow leaving it at the default ratio but merely setting it to off makes more sense to me.  Unless/until we change the UI to remove that checkbox as discussed elsewhere (and even then, probably no need to do anything different).

Reply:
Sorry, I seem to have misunderstood this: it looks like the first `0` argument to `setSwingParameters` is what disables swing here. Could you please add a comment then to make it more clear?

Reply:
like setting it to `false` maybe?

Reply:
It isn't a bool variable, it's `0`, or `MScore::division / 2`, or `MScore::division / 4`.

Reply:
@dmitrio95 Done.